pyproject.toml

```
@@ -50,7 +49,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9.2, !=3.9.7"
+python = ">=3.10,<3.13"
```

```suggestion
python = ">=3.10"
```
nit: lets not define an upper bound restriction here, this will allow others to experiment with 3.13

---

sure , thanks!